### ISSUE TITLE
Kabat numbering not working with --details ?

### ISSUE THREAD

**Issue by ProustIdee** [antigenomics/arda#44]
./migmap --use-kabat --blast-dir /data/xxx/ncbi-igblast-1.6.1/bin --data-dir /data/xxx/migmap-1.0.2/data -R IGH -S human --by-read --details fr1aa,fr2aa,fr3aa,cdr1aa,cdr2aa test.txt test.kabat.txt
[Tue Mar 07 12:10:17 PST 2017 MIGMAP +00m00s] Loaded database for human IGH gene(s): 89 Variable, 37 Diversity and 6 Joining segments.
[Tue Mar 07 12:10:17 PST 2017 MIGMAP +00m00s] Creating temporary BLAST database /data/xxx/migmap-1.0.2/data/database-a06324d2-8746-4ea1-9bd3-35ae88c29cec.
[Tue Mar 07 12:10:17 PST 2017 MIGMAP +00m00s] Annotating variable segments.
[Tue Mar 07 12:10:19 PST 2017 MIGMAP +00m02s] Fully annotated 0 of 89 segments.
[Tue Mar 07 12:10:19 PST 2017 MIGMAP +00m02s] Analyzing sample test.txt
[Tue Mar 07 12:10:19 PST 2017 MIGMAP +00m02s] Started analysis
[Tue Mar 07 12:10:19 PST 2017 MIGMAP +00m02s] Loaded 159 reads.
[Tue Mar 07 12:10:24 PST 2017 MIGMAP +00m07s] Finished analysis. Processed 2000 reads, of them 100.0% mapped. Among mapped reads 42.2% passed quality filter and 29.8% passed all filters.
[Tue Mar 07 12:10:24 PST 2017 MIGMAP +00m07s] Finished.

./migmap -Xmx14G --blast-dir /data/xxx/ncbi-igblast-1.6.1/bin --data-dir /data/xxx/migmap-1.0.2/data -R IGH -S human --by-read --details fr1aa,fr2aa,fr3aa,cdr1aa,cdr2aa test.txt test.imgt.txt
[Tue Mar 07 12:10:54 PST 2017 MIGMAP +00m00s] Loaded database for human IGH gene(s): 89 Variable, 37 Diversity and 6 Joining segments.
[Tue Mar 07 12:10:54 PST 2017 MIGMAP +00m00s] Creating temporary BLAST database /data/xxx/migmap-1.0.2/data/database-3672583a-048e-4a01-9719-034d9c5a541d.
[Tue Mar 07 12:10:54 PST 2017 MIGMAP +00m00s] Annotating variable segments.
[Tue Mar 07 12:10:57 PST 2017 MIGMAP +00m02s] Fully annotated 89 of 89 segments.
[Tue Mar 07 12:10:57 PST 2017 MIGMAP +00m02s] Analyzing sample test.txt
[Tue Mar 07 12:10:57 PST 2017 MIGMAP +00m02s] Started analysis
[Tue Mar 07 12:10:57 PST 2017 MIGMAP +00m02s] Loaded 393 reads.
[Tue Mar 07 12:11:02 PST 2017 MIGMAP +00m07s] Finished analysis. Processed 2000 reads, of them 100.0% mapped. Among mapped reads 42.2% passed quality filter and 30.0% passed all filters.
[Tue Mar 07 12:11:02 PST 2017 MIGMAP +00m07s] Finished.

When comparing test.imgt.details to test.kabat.details , the fr1aa, fr2aa, fr3aa, cdr1aa, and cdr2aa sequences at the end of the tables are messed up (concatenated together, and seemingly wrong sequences).

IMGT on read SRR747768.2037:
`XXQLVESGGGVVQPGRSLRLSCAAS       MHWVRQAPGKGLEWVAV       YYADSVKGRFTISRDNSKNTLYLQMNSLRAEDTAVYY   GFTFSSYG        ISYDGSNK`

Kabat on read SRR747768.2037:
`XXAAGGVWGRRGPAWEVPETLLCSLWIHLQ*LWHALGPPGSRQGAGVGGSYII*WK**ILCRLREGPIHHLQRQFQEHAVSANEQPES*GHGCVL`

[test.txt](https://github.com/mikessh/migmap/files/825693/test.txt)
[test.imgt.txt](https://github.com/mikessh/migmap/files/825698/test.imgt.txt)
[test.kabat.txt](https://github.com/mikessh/migmap/files/825697/test.kabat.txt)

**Comment by Yeli-bioinformatics** [comment]
Yes, I also notice this issue, any update how to solve this problem? thanks.

**Comment by ProustIdee** [comment]
I ended up manually parsing the IMGT output.
